timeseries/src/autogluon/timeseries/models/chronos/pipeline/chronos_bolt.py

```diff
@@ -528,3 +528,65 @@ def from_pretrained(cls, *args, **kwargs):
 
         model = class_.from_pretrained(*args, **kwargs)
         return cls(model=model)
+
+
+def patch_chronos_bolt_output_quantiles(model: ChronosBoltModelForForecasting, new_quantiles: list[float]) -> None:
```

Comment:
To discuss: Should this move to an instance method of `ChronosBoltModelForForecasting` or `ChronosBoltPipeline`?